src/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -331,14 +331,8 @@
     <Reference Include="System.Threading.Thread" />
     <Reference Include="System.Transactions.Local" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'uapaot'">
+  <ItemGroup>
     <Reference Include="System.Xml.ReaderWriter" />
     <Reference Include="System.Xml.XmlSerializer" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'">
```

Comment:
The UAPAOT path here that did not depend on private reflection is better. Should we rather switch over to the uapaot path unconditionally?

Reply:
I don't have any context here, should this be done as part of this PR?

Reply:
It is basically about picking the uapaot path here and in `SqlXml.cs`; and deleting the !uapaot path instead. If you do not feel comfortable doing it, you can leave it for later.